akka-remote/src/test/scala/akka/remote/artery/AeronStreamsApp.scala

```diff
@@ -157,20 +159,21 @@ object AeronStreamsApp {
     var t0 = System.nanoTime()
     var count = 0L
     var payloadSize = 0L
-    Source.fromGraph(new AeronSource(channel1, streamId, aeron, taskRunner))
-      .map { bytes ⇒
-        r.onMessage(1, bytes.length)
-        bytes
+    Source.fromGraph(new AeronSource(channel1, streamId, aeron, taskRunner, pool))
+      .map { envelope ⇒
+        r.onMessage(1, envelope.byteBuffer.limit)
+        envelope
```

Comment:
I am not sure the changes in this file are correct, flipping is missing so I need to double-check this.